apps/condo/pages/ticket/[id]/index.tsx

```diff
@@ -106,6 +109,20 @@ export const TicketFileList: React.FC<ITicketFileListProps> = ({ files }) => {
         a.click()
     }
 
+    const handleFileDownload: (file: UploadFile) => void = async (file: UploadFile) => {
+        const forbiddenFileTypes = ['html', 'svg', 'txt']
```

Comment:
this is static contant, we use to name them in caps like ALLOWED_FILE_TYPES and move outside of functions to be able to reuse

Reply:
Redesigned using Regex